docs/Course Wiki/PHYS Course Offerings/PHYS212.md

```diff
@@ -0,0 +1,59 @@
+
+## Prerequisites
+
+The official prerequisite for PHYS212 is PHYS211. 
+
+## When to Take It
+
+ECE majors typically take PHYS212 in their freshman year, most commonly in spring after doing PHYS211 in the fall. It is a course that is quite helpful for ECE 110 and 210 if the student understands the concepts properly. The student will then find that it provides a nice scientific basis and intuition for a lot of ideas in 110 and 210 that are merely presented axiomatically as equations to be used in circuit analysis. 
+
+## Course Structure
+
+PHYS212 holds 2 50-minute lectures per week, with prelectures and checkpoints to complete before each lecture. Attendance is graded at lectures via the iClicker platform. Nearly all of the material is covered in the prelectures, albeit in a more concise and summarized manner. Lectures thus provide an excellent opportunity to ask questions and practice concepts via interactive iClicker questions.
+Homework is assigned after each lecture in smartPhysics, which does cost money. The homework for each week's lectures are typically due at the same time early during the following week.
+There is a weekly 2-hour discussion that involves solving a problem set and taking a short quiz on the previous week’s material
+There are 2 midterm exams and 1 final exam, all taken through Prarielearn at the CBTF.
+There are weekly labs that are each 2 hours long, where you use the IOLab and its E&M kit (that you need to purchase for yourself) to complete laboratory exercises and submit a report with your group at the end of every class. Teamwork is heavily emphasized in the PHYS 212 Lab. 
+
+## Instructors
+
+In the past, this course has been taught by Professors Mahmood and Kuo.
+
+## Course Tips
+
+This course at times can get a little abstract compared to PHYS211. Trying one’s best to get an intuitive grasp of such concepts in a non-visual way (the author’s high school physics teacher used to think of electric potential energy as how angry electrons are in a particular area, among other such analogies). Make sure to take detailed notes and work out problems done in lecture. Also, definitely make sure to understand every homework problem, as they do a great job of testing conceptual understanding and problem-solving creativity. The discussion quizzes are fairly easy, don’t worry too much about them. Just remember to participate and attend everything, as all course meetings (lecture, lab, attendance) have graded attendance. In terms of math knowledge, most mathematics knowledge beyond the very basic ideas of differentiation and integration are merely presented in derivations of equations and never used in actual computation.
+
+## Life After
+
+This course, as mentioned before, helps a lot in understanding concepts in electronics and electromagnetism courses such as ECE 110, 210, and 330. 
```

Comment:
Mention PHYS214, also talk about ECE329 and ECE350.  Also briefly explain what concepts will be used again in these courses, that is, why would a student who enjoyed PHYS212 like ECE210 or ECE329?